lib/decidim/time_tracker/reports_engine.rb

```diff
@@ -5,23 +5,22 @@ module TimeTracker
     # This is the engine that runs on the global space for `decidim-time_tracker`.
     # It handles stuff unrelated to participatory spaces or components
     # Such as user space managemt
-    class DirectoryEngine < ::Rails::Engine
-      isolate_namespace Decidim::TimeTracker::Directory
+    class ReportsEngine < ::Rails::Engine
+      isolate_namespace Decidim::TimeTracker::Reports
 
       paths["db/migrate"] = nil
       paths["lib/tasks"] = nil
 
       routes do
         authenticate(:user) do
-          resources :voluntary_work, controller: "voluntary_work"
-          root to: "voluntary_work#index"
+          root to: "user#index"
         end
       end
 
       initializer "decidim.user_menu" do
         Decidim.menu :user_menu do |menu|
-          menu.item t("voluntary_work", scope: "layouts.decidim.user_profile"),
-                    decidim_time_tracker.voluntary_work_index_path,
+          menu.item t("time_tracker", scope: "layouts.decidim.user_profile"),
+                    decidim_time_tracker.root_path,
```

Comment:
Is this correct? shouldn't this point to the users controller?

Reply:
For some reason it is the only way I found to point to the correct route for the user's voluntary work (`/timetracker`)

Reply:
We can change it later if we find a more elegant solution